readme.txt

```diff
@@ -34,6 +34,43 @@ make
 then as root
 make install
 
+
+Customize Desktop Environment for xRDP Session
+
+If you do not want to use the default desktop environment, you can customize it by creating a .Xclients file (X is capital!!!) in your home directory to launch the desktop environment you want and making it executable. In order to do this, open a terminal and run one of the following commands
+
+Gnome 3:
+echo "gnome-session" > ~/.Xclients
+chmod +x ~/.Xclients
```

Comment:
I'm not sure `.Xclients` is correct, if a DM is used (`xdm`, `lightdm`...), the file to be used is `~/.xsession`. If the session is started from console, it has to be `.xinitrc`